pkgs/applications/misc/coltrane/default.nix

```
@@ -0,0 +1,22 @@
+{ lib
+, bundlerApp
+, bundlerUpdateScript
+}:
+
+bundlerApp rec {
+  pname = "coltrane";
+  gemdir = ./.;
+  exes = [ "coltrane" ];
+
+  meta = with lib; {
+    homepage = "https://github.com/pedrozath/coltrane";
+    description = "A music calculation library/CLI";
+    longDescription = ''
+      coltrane allows to search for Notes, Chords, Scales for
+      guitar, bass, piano and ukelele
+    '';
+    license = licenses.mit;
+  };
+
+  passthru.updateScript = bundlerUpdateScript pname;
```

```suggestion
  passthru.updateScript = bundlerUpdateScript pname;

  meta = with lib; {
    homepage = "https://github.com/pedrozath/coltrane";
    description = "A music calculation library/CLI";
    longDescription = ''
      coltrane allows to search for Notes, Chords, Scales for
      guitar, bass, piano and ukelele
    '';
    license = licenses.mit;
  };
```
meta should be last according to the contributing guide